homeassistant/components/goodwe/sensor.py

```diff
@@ -101,7 +108,8 @@ def _get_sensor_description(sensor):
     if sensor.unit == "%" and sensor.kind == SensorKind.BAT:
         desc.state_class = STATE_CLASS_MEASUREMENT
         desc.device_class = DEVICE_CLASS_BATTERY
-
+    if sensor.id_ not in _MAIN_SENSORS:
+        desc.entity_category = "diagnostic"
```

Comment:
This should be `ENTITY_CATEGORY_DIAGNOSTIC`

Reply:
Done